src/components/breadcrumbs/crumb/crumb.style.ts

```diff
@@ -3,35 +3,50 @@ import Link, { LinkProps } from "../../link";
 
 interface StyleCrumbProps extends LinkProps {
   isCurrent?: boolean;
+  isDarkBackground: boolean;
 }
 
+const getCurrentCrumbColor = (isDarkBackground: boolean) => {
+  return isDarkBackground
+    ? "var(--colorsUtilityYang100)"
+    : "var(--colorsUtilityYin090)";
+};
+
 export const StyledCrumb = styled(Link)<StyleCrumbProps>`
   font: var(--typographyLinkTextM);
-  ${({ isCurrent }) =>
+  ${({ isCurrent, isDarkBackground }) =>
     isCurrent &&
     css`
       a {
-        color: var(--colorsUtilityYin090);
+        color: ${getCurrentCrumbColor(isDarkBackground)};
         text-decoration: none;
         font: var(--typographyBreadcrumbCurrentPageM);
-        cursor: text;
         :hover {
-          color: var(--colorsUtilityYin090);
+          color: ${getCurrentCrumbColor(isDarkBackground)};
           text-decoration: none;
           cursor: text;
         }
       }
     `}
 `;
 
-export const Divider = styled.span.attrs({
-  children: "/",
-  "aria-hidden": "true",
-})`
-  margin: 0px var(--spacing050) 0px var(--spacing100);
-  line-height: 16px;
-  font: var(--typographyBreadcrumbSeparatorM);
-  color: var(--colorsUtilityYin055);
+interface DividerProps {
+  isDarkBackground: boolean;
+}
+
+export const Divider = styled.span<DividerProps>`
+  ::after {
+    content: "/";
+    margin: 0px var(--spacing050) 0px var(--spacing100);
```

Comment:
**nitpick/suggestion(non-blocking):** The design token for `0px` is `spacing000`. I'm happy if you leave it as is or want to change it to a token value.